src/commands/build.ts

```diff
@@ -0,0 +1,44 @@
+import { Command, flags } from '@oclif/command';
+import { build, optimize } from '../lib/deployment';
+
+export default class Build extends Command {
+  static description = 'Build and optimize wasm bytecode.';
+
+  static flags = {
+    // "no-optimize": flags.boolean({
+    //   description: "do not optimize the wasm.",
+    //   default: false,
+    // }),
+    optimize: flags.boolean({
+      description: 'optimize the wasm after the build.',
+      default: false,
```

Comment:
```suggestion
    optimize: flags.boolean({
      description: 'optimize the wasm after the build.',
      default: true,
```

I think we should have optimize set to true by default. What's the situation where you build and not deploy?